algorithms/unit_tests/CMakeLists.txt

```diff
@@ -153,6 +153,18 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;HIP;SYCL;OpenMPTarget)
       list(APPEND STDALGO_SOURCES_E Test${Name}.cpp)
     endforeach()
 
+    # ------------------------------------------
+    # std team P
+    # ------------------------------------------
+    set(STDALGO_TEAM_SOURCES_P)
+    foreach(Name
+  StdAlgorithmsCommon
+  StdAlgorithmsTeamExclusiveScan
+  StdAlgorithmsTeamTransformExclusiveScan
```

Comment:
The indentation looks weird. You are using tabs anywhere else.

Reply:
that was a side effect: i lost all my settings on the sandia machine, so I had to use an editor that i don't use regularly :)
fixed now